layers/+tools/meson/README.org

```diff
@@ -0,0 +1,28 @@
+#+TITLE: meson layer
+#+TAGS: layer|tool
+
+[[file:img/meson.png]]
+
+* Table of Contents                                        :TOC_4_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#key-bindings][Key bindings]]
+
+* Description
+This layer adds support [[https://mesonbuild.com/][Meson]] scripts.
+
+** Features:
+- Support for meson build scripts through meson-mode.
+- Syntax highlighting.
+
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =meson= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+* Key bindings
+
+| Key Binding | Description                              |
+|-------------+------------------------------------------|
+| ~F1~        | Lookup documentation for thing at point. |
```

Comment:
I know the <kbd>F1</kbd> comes from meson-mode itself but IMO spacemacs could use more spacemacs-y keybinding, along the lines of <kbd>SPC h d d</kbd> (seems free?) or <kbd>SPC a z d</kbd> (dash-at-point)

Reply:
Should this maybe be a leader key for the mode such as `SPC m h`?

Reply:
I only suggest as `f1` binds to `meson-lookup-doc-at-point` which seems major-mode specific.

Reply:
I would also vote for a major mode specific key, per convention this would then be `SPC m h h`